src/components/CanvasWrapper.tsx

```diff
@@ -261,13 +261,13 @@ export default function CanvasWrapper(inputProps: CanvasWrapperProps): ReactElem
   useMemo(() => {
     const annotationLabels = props.annotationState.data.getLabels();
     const timeToAnnotationLabelIds = dataset ? props.annotationState.data.getTimeToLabelIdMap(dataset) : new Map();
+    canv.isAnnotationVisible = props.annotationState.visible;
```

Comment:
Fixes a bug where the canvas would not render the annotations until the next update